softgroup/services/main-router/src/main/java/com/softgroup/main/router/configuration/MainRouterAppCfg.java

```diff
@@ -0,0 +1,14 @@
+package com.softgroup.main.router.configuration;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * @author AlexKazmirchuk
+ * @since 28.02.17.
+ */
+
+@Configuration
+@ComponentScan("com.softgroup")
```

Comment:
@AlexKazmirchuk Temporary is not allowed in PR. If you need something temporary - keep it in your local repository.